provers/celestia-prover/README.md

```diff
@@ -4,15 +4,15 @@ The Celestia Prover is a gRPC service that generates zero-knowledge proofs for C
 
 ## Usage
 
-> [!WARNING]
+> ⚠️ **Warning**
```

Comment:
[nit] the previous renders nicely on Github but I'm fine w/ either

Ref: https://github.com/orgs/community/discussions/16925